meson.build

```diff
@@ -4004,6 +4004,20 @@ public_programs += executable(
         install_rpath : rootpkglibdir,
         install : true)
 
+if conf.get('HAVE_QRENCODE') == 1
+        public_programs += executable(
+                'systemd-bsod',
+                'src/journal/bsod.c',
+                include_directories : includes,
+                link_with : [libshared],
+                dependencies : [userspace,
+                                versiondep,
+                                libqrencode],
+                install_rpath : rootpkglibdir,
+                install_dir : rootlibexecdir,
+                install : true)
+endif
```

Comment:
@bluca , is this it sir?

Reply:
yes that's fine

Reply:
> yes that's fine

Okay sir.

I'll now work on the alignment fix.